app/build.gradle

```diff
@@ -22,7 +22,7 @@ dependencies {
         transitive=true
     }
 
-
+    implementation "com.android.support:customtabs:$SUPPORT_LIB_VERSION"
```

Comment:
Using a custom tab here seems unnecessary to me. We can simply launch the browser to maintain consistency about how web URLs are handled by the app. You could open a separate discussion for using custom tabs if its really good. :)

Reply:
I had created an issue for that sir.
Please review it once.
https://github.com/commons-app/apps-android-commons/issues/1143

Reply:
Aah! I missed it earlier. Yeah, its usage across the app seems a good option.